packages/kbn-securitysolution-list-utils/src/helpers/index.ts

```diff
@@ -1021,3 +1025,21 @@ export const getMappingConflictsInfo = (field: DataViewField): FieldConflictsInf
   }
   return conflicts;
 };
+
+/**
+ * Given an exceptions list, determine if any entries have an "IS" operator with a wildcard value
+ */
+export const getHasWrongOperator = (items: ExceptionsBuilderReturnExceptionItem[]): boolean => {
+  let hasWrongOperator = false;
+  items[0]?.entries.forEach((e) => {
```

Comment:
Would you consider using arraych.some() instead of foreach? :)

Reply:
this is actually a great idea